konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt

```diff
@@ -147,6 +148,15 @@ class ParticleSystem(private val konfettiView: KonfettiView) {
         return this
     }
 
+    /**
+     * Set bottom gravity
+     * If the value is negative it will be set to default value.
+     */
+    fun setGravity(y: Float): ParticleSystem {
+        gravity.y = if (y < 0) 0.01f else y
```

Comment:
Minor suggestion:

```Kotlin
gravity.y = y.coerceAtLeast(0.01f)
```

Reply:
Updated code as suggested.

Reply:
> @akashraghav thank you for your contribution. Could you run `./gradlew spotlessApply` to make sure the CI succeeds?

I did this earlier and pushed the change reported in log earlier, you can view diff `konfetti/src/main/java/nl/dionsegijn/konfetti/Confetti.kt` but the CI check is still reporting same issue. While, I'm getting build success locally on both `./gradlew spotlessCheck` and `./gradlew spotlessApply` calls. Is there a way to trigger check again?

```
./gradlew spotlessCheck

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 4s
5 actionable tasks: 5 executed

```

```
./gradlew spotlessApply

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 926ms
5 actionable tasks: 1 executed, 4 up-to-date
```